scripts/sct_pipeline_makefig.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+#
+# Generate figure from the output of sct_pipeline.
+#
+# Copyright (c) 2013 Polytechnique Montreal <www.neuro.polymtl.ca>
```

Comment:
2018

Reply:
fixed in 9d77c82139345e4600a708702d8ba06729c7a1f6